core/tsconfig.json

```diff
@@ -15,7 +15,8 @@
       "dom.iterable",
       "es2017",
       "es2020",
-      "es2021"
+      "es2021",
+      "esnext"
```

Comment:
This is required otherwise we will get a type error on build: https://github.com/microsoft/playwright/issues/28262